docs/04-documentation.md

```diff
@@ -0,0 +1,46 @@
+---
+title: How to update documentation on ECL
+label: Edit documentation
+---
+
+This page explains how to update documentation directly on GitHub.
+
+## Prerequisite
+
+If you don't already have a GitHub account, make sure to create one and log in with it.  
+You also have to be part of team [ECL doc editors](https://github.com/orgs/ec-europa/teams/ecl-doc-editors/).
+
+## Step 1: Open documentation file
+
+- Go to [ECL repository](https://github.com/ec-europa/europa-component-library)
+- Locate the component you wish to edit. Components are located in `framework/components`
+- Open the README.md file in that component (example: `framework/components/ecl-footers/README.md`)
+
+<img src="/assets/screenshots/git_edit.png" alt="GitHub edit" width="800" />
+
+## Step 2: Edit the documentation
+
+- Click on the edit button (pencil icon on top right)
+- Make any modification you wish by using markdown editor. Here is a [cheatsheet for markdown syntax](https://guides.github.com/features/mastering-markdown/)
+- You can see result directly by switching to the "Preview" tab
```

Comment:
Or by checking netlify :)

Reply:
Yes, but let's not make the doc more complex :)

Reply:
Agree, it can be a further improvement